src/v2/Components/NavBar/Menus/__tests__/NavBarUserMenu.jest.tsx

```diff
@@ -37,6 +43,22 @@ describe("NavBarUserMenu", () => {
     expect(wrapper.find("button").last().text()).toContain("Log out")
   })
 
+  it('render "Alerts" menu item only when ENABLE_SAVED_SEARCH flag is enabled', () => {
+    sd.ENABLE_SAVED_SEARCH = true
+
+    const wrapper = getWrapper()
+    const links = wrapper.find("a")
+
+    expect(links.map(a => [a.prop("href"), a.text()])).toEqual([
+      // Label also includes SVG image title
+      ["/user/purchases", "Pending Order History"],
+      ["/user/saves", "Save Saves & Follows"],
+      ["/user/alerts", "Watch lot Alerts"],
```

Comment:
**question**: is the title of "Watch lot Alerts" correct here?

Reply:
Yes, because it also includes SVG image title (see the comment above)